### PR TITLE
Pass in all the data from the streams to the textfield to allow it ac…

### PR DESCRIPTION
**Pull request by bloveless**
…cess to its siblings data. I've only added the data to the parser in the textfield. I'm sure it could be used in other places field_types.

### CODE REVIEW

File: system/cms/modules/streams_core/field_types/textarea/field.textarea.php [hunk 1]
```diff
@@ -86,11 +86,18 @@ public function pre_output($input, $params)
 			return $input;
 		}
 
+		// allow additional data to be passed to the parser
+		$parser_data = array();
+		if(!empty($params['parser_data']))
+		{
+			$parser_data = $params['parser_data'];
+		}
+
```
Comment (RyanThompson) [inline system/cms/modules/streams_core/field_types/textarea/field.textarea.php]: Good good good

File: system/cms/modules/streams_core/models/row_m.php [hunk 2]
```diff
@@ -1360,7 +1360,7 @@ public function format_row($row, $stream_fields, $stream, $return_object = true,
 	 * @access 	public
 	 * @params
 	 */
-	public function format_column($column_slug, $column_data, $row_id, $type_slug, $field_data, $stream, $plugin = true)
+	public function format_column($column_slug, $column_data, $row_id, $type_slug, $field_data, $stream, $plugin = true, $stream_data = array())
```
Comment (RyanThompson) [inline system/cms/modules/streams_core/models/row_m.php]: Don't you just HATE non OOP stuff anymore?

**Comment by RyanThompson** [comment]
Nice work

File: system/cms/modules/streams_core/models/row_m.php (hunk 2 above)
Comment (bloveless) [inline system/cms/modules/streams_core/models/row_m.php]: Hate it is an understatement :)

**Pull request merged**